src/compass/utils/h5_helpers.py

```diff
@@ -3,10 +3,16 @@
 import isce3
 import numpy as np
 from osgeo import osr
+from s1reader.s1_reader import is_eap_correction_necessary
 
 import compass
+from compass.utils.lut import compute_geocoding_correction_luts
 from compass.utils.raster_polygon import get_boundary_polygon
 
+
+TIME_STR_FMT = '%Y-%m-%d %H:%M:%S'
```

Comment:
based on your response, I think you would have meant `'%Y-%m-%d %H:%M:%S.%f'`?

Reply:
Agreed. I will make that change. Will push that along with overhaul of group/dataset hierarchy.